src/views/RoomView.vue

```diff
@@ -109,46 +130,50 @@ const showHide = computed({
 .icon-button, .timer {
   padding: 0px 6px;
 }
+.actionable {
+  cursor: pointer;
+  transform: scale(1);
+  z-index: 5;
+}
+.actionable:hover {
+  /* effect like taking the card */
+  transform: scale(1.1) rotateX(-5deg) translateY(-20px);
+  box-shadow: 0 -5px 5px 5px #0008;
+  z-index: 10;
+}
+.mine.actionable:hover {
+  /* effect like pulling the card */
+  transform: rotateX(5deg) translateY(20px);
+}
 .selected {
   opacity: 0;
-  transform: translate(0, 0);
-  animation-name: moveCard;
-  animation-duration: 1s;
-}
-@keyframes moveCard {
-  0% {
-    opacity: 1;
-    transform: translate(0, 0);
-  }
-  100% {
-    opacity: 0;
-    transform: translate(0, -100px);
-  }
+  transform: translateY(-200px) rotateX(-10deg) scale(1.3);
+  box-shadow: 0 -15px 15px 15px #0000;
 }
-
-@keyframes cardSet {
-  0% {
-    opacity: 0;
-    transform: translate(0, 100px);
-  }
-  100% {
-    opacity: 1;
-    transform: translate(0, 0);
-  }
+.unset {
+  opacity: 0;
+  transform: translateY(200px) rotateX(-20deg) scale(1.5);
```

Comment:
```suggestion
  transform: translateY(20px) rotateX(-20deg) scale(1.5);
```
c'est ce translateY à 200px qui ajoute le scrollbar

Reply:
ma suggestion n'est pas vraiment la solution puisque ça modifie l'animation...